src/slic3r/Utils/PresetUpdater.cpp

```diff
@@ -818,9 +818,9 @@ void PresetUpdater::priv::sync_tooltip(std::string http_url, std::string languag
         fs::path cache_root = fs::path(data_dir()) / "resources/tooltip";
         try {
             auto vf = cache_root / "common" / "version";
-            if (fs::exists(vf)) load_string_file(vf, common_version);
+            if (fs::exists(vf)) fs::load_string_file(vf, common_version);
```

Comment:
This will very like to break the build as Boost 1.8.4 has removed this function.
And Orca has migrated to Boost 1.8.4 recently.
So now we have re-implemented this function in src/libslic3r/Utils.hpp:661
We should revert this change

Reply:
1.84?

Reply:
https://www.boost.org/users/history/version_1_84_0.html

Reply:
That's what I thought.